Fix GH Workflow Action for rendering

Get:31 http://azure.archive.ubuntu.com/ubuntu noble-security/universe Translation-en [218 kB]
Get:32 http://azure.archive.ubuntu.com/ubuntu noble-security/universe amd64 Components [74.3 kB]
Get:33 http://azure.archive.ubuntu.com/ubuntu noble-security/universe amd64 c-n-f Metadata [20.6 kB]
Get:34 http://azure.archive.ubuntu.com/ubuntu noble-security/restricted amd64 Packages [2599 kB]
Get:35 http://azure.archive.ubuntu.com/ubuntu noble-security/restricted Translation-en [600 kB]
Get:36 http://azure.archive.ubuntu.com/ubuntu noble-security/restricted amd64 Components [212 B]
Get:37 http://azure.archive.ubuntu.com/ubuntu noble-security/multiverse amd64 Components [208 B]
Fetched 14.8 MB in 2s (8681 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
Package libgl1-mesa-glx is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'libgl1-mesa-glx' has no installation candidate
Error: Process completed with exit code 100.

--- a/.github/workflows/render_models.yml
+++ b/.github/workflows/render_models.yml
@@ -18,7 +18,7 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y xvfb unzip curl libgl1-mesa-glx libglu1-mesa
+          sudo apt-get install -y xvfb unzip curl libgl1 libglu1-mesa
 
       - name: Install LDraw parts library
         run: |

--- a/ROADMAP.md
+++ b/ROADMAP.md
@@ -12,6 +12,7 @@
 - [x] Set up GitHub Actions for automated LEGO model rendering
 - [x] Implement LDR model for basic NAND gate (`sg13g2_nand2_1`)
 - [x] Implement LDR model for basic NOR gate (`sg13g2_nor2_1`)
+- [x] Fix GitHub Action rendering by updating OpenGL dependencies
 
 ## Next 5 Steps
 - [ ] Implement LDR model for D-Flip-Flop (`sg13g2_dfrbp_1`)

